buildcontext/resolver.go

```diff
@@ -83,6 +83,17 @@ func NewResolver(cleanCollection *cleanup.Collection, gitLookup *GitLookup, cons
 	}
 }
 
+// ExpandWildcard will expand a wildcard BUILD target in a remote Git
+// repository. The pattern is the path relative to the target path and should be
+// in the form 'my/path/*'
+func (r *Resolver) ExpandWildcard(ctx context.Context, gwClient gwclient.Client, platr *platutil.Resolver, target domain.Target, pattern string) ([]string, error) {
+	if !target.IsRemote() {
+		return nil, errors.Errorf("unexpected local reference %s", target.String())
```

Comment:
Should we just move the implementation for the local case over here too, for symmetry?

Reply:
Sure, sounds good. I wasn't certain about where the new code belongs.